config/ingest.go

```diff
@@ -41,6 +41,10 @@ type Ingest struct {
 	// IngestWorkerCount sets how many ingest worker goroutines to spawn. This
 	// controls how many concurrent ingest from different providers we can handle.
 	IngestWorkerCount int
+	// MinimumKeyLengt causes any multihash, that has a digest length less than
+	// this, to be ignored. If using storethehash, this value is automatically
+	// set to 4 if it was configured to be anything less.
+	MinimumKeyLength int
```

Comment:
This will default to `0` right? 

Should we explicitly set its value across instance configs to `4`?

Reply:
Side note and out of the scope of this PR: we probably want a maximum size config too to limit malicious behaviour.